packages/postgres-database/Makefile

```diff
@@ -121,7 +121,7 @@ migrate: $(DOT_ENV_FILE) ## basic migration update (use ONLY for development pur
 
 .PHONY: up-pg down-pg up-prod down-prod
 docker-compose-configs = $(wildcard tests/docker-compose*.yml)
-up-pg up-prod: $(docker-compose-configs) ## starts pg server
+up-pg up-prod: $(docker-compose-configs) ## starts pg server. Use POSTGRES_DATA_VOLUME to pass volume mount
```

Comment:
actually since we use ```setup-prod``` and ```setup-commit``` I am not sure that target is really needed. and setup-prod has a guard-, which you could also put on ```up-prod``` right?

Reply:
AFAIK setup-commit is to prepare for migration. Here i just want to use the docker-compose to create a stack with adminer and postpro and load the volume that i copied from master.